### DISCUSSION TITLE
Conceito de "retorno" - Fundamentos de C#

### DISCUSSION THREAD

**Discussion by fjosematos** [balta-io/forum#607] [Tira Dúvidas] [answered]
Paira sempre uma duvida nesse conceito pois sempre há a explicação de que 'VOID" não retorna nada. No caso do exemplo há a impressão na tela "C# é legal". Isso não seria um tipo de retorno?

**Comment by henriqueweege** [comment, marked as answer]
Oi, @fjosematos ! Tudo bem?
Quando se fala de _retorno_ em programação estamos nos referindo ao que é retornado por uma função. 
Por exemplo, você tem como passar para a função "Max" da class "Math" dois inteiros, e essa função vai te _retornar_ o maior deles. Em outras palavras, _retorno_ é o valor que a função te devolve depois de ser chamada, independentemente de outros efeitos que ela possa gerar.
No caso do método "WriteLine" da classe "Console", você tem o efeito gerado na tela, mas a função te retorna _void_, que é vazio, ou seja, valor nenhum ;)

Espero ter ajudado! Valeeeu!
